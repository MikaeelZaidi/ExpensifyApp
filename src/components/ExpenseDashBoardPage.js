import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListItem from './ExpenseListItem';
import ExpenseListFilter from './ExpenseListFilter';
import { connect } from 'react-redux';

const ExpenseDashBoardPage = (props) => {
    return (
        <div>
            <ExpenseListFilter />
            <ExpenseList />


        </div>
    )
}


export default ExpenseDashBoardPage