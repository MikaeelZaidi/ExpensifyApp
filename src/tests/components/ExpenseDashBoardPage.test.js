import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import ExpenseDashBoardPage from '../../components/ExpenseDashBoardPage'


test('Should test the Expense Dashboard Page component',()=>{
    const wrapper = shallow(<ExpenseDashBoardPage/>)
    expect(toJSON(wrapper)).toMatchSnapshot()
})