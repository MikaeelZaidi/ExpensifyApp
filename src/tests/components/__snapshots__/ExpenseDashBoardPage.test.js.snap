// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test the Expense Dashboard Page component 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <ExpenseSummary />
  <Connect(ExpenseList) />
</div>
`;
